docs/ref/forms/api.txt

```diff
@@ -552,15 +592,32 @@ The template used to render a field's ``<label>``, used when calling
 form by overriding this attribute or more generally by overriding the default
 template, see also :ref:`overriding-built-in-form-templates`.
 
+Output styles
+-------------
+
+As well as rendering the form with ``print(form)``, or in a template
```

Comment:
Likely this is elsewhere in the docs, but do folk actually use `print()`. I'd expect folk to be using `str()` or a `str()` like method (i.e. `mark_safe()`).